### PR TITLE
[ DP-27]  테스트 환경 설정 및 코드 작성

### PR DESCRIPTION
**Pull request by mandelina**

### CODE REVIEW

**Comment by minyoung22222** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>5f4f33b</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://devdevdev-client-h58nutfzu-minyoung22222s-projects.vercel.app'>https://devdevdev-client-h58nutfzu-minyoung22222s-projects.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/minyoung22222s-projects/devdevdev-client/jfLy7aV9mugC23WjSRT5ERkBv2Tc'>https://vercel.com/minyoung22222s-projects/devdevdev-client/jfLy7aV9mugC23WjSRT5ERkBv2Tc</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/dreamyPatisiel/devdevdev-client/actions/runs/7600708125)

File: pages/login/LoginButton.1.tsx
Comment (minyoung22222) [inline pages/login/LoginButton.1.tsx:1]: 이 버튼은 안 쓰는 거면 삭제할까욧?!?

**Review by minyoung22222 — COMMENTED** [review]
이거 우선 이대로 push 하고 로그인 할 때 수정 하는것 어때용??

**Review by mandelina — COMMENTED** [review]
고생하셨숩니다~~ mockServiceWorker?는 좀 더 봐봐야겠네욤 🤔

**Review by mandelina — COMMENTED** [review]
??

**Review by minyoung22222 — APPROVED** [review]
수고하셨씁니다~~

**Pull request merged**